soh/soh/GameMenuBar.cpp

```diff
@@ -264,15 +264,15 @@ namespace GameMenuBar {
                         currentFps = 60;
                     }
                     CVarSetInteger("gInterpolationFPS", currentFps);
-                    SohImGui::RequestCvarSaveOnNextTick();
+                    ImGui::RequestCvarSaveOnNextTick();
```

Comment:
https://github.com/Kenix3/libultraship/blob/d63e527423e40404d57682fd0384389428b9ff81/src/menu/ImGuiImpl.cpp#L982-L984

this is in the ship namespace in there

https://github.com/Kenix3/libultraship/blob/d63e527423e40404d57682fd0384389428b9ff81/src/menu/ImGuiImpl.cpp#L86

so this should just be

```suggestion
                    Ship::RequestCvarSaveOnNextTick();
```

Reply:
this isn't causing a build failure because it's ifdef'd behind wii u

Reply:
after testing wii u and switch builds in https://github.com/HarbourMasters/Shipwright/pull/2823, this was the only SoH change needed